cmd/ipfs-cluster-service/main.go

```diff
@@ -343,12 +368,18 @@ snapshot to be loaded as the cluster state when the cluster peer is restarted.
 If an argument is provided, cluster will treat it as the path of the file to
 import.  If no argument is provided cluster will read json from stdin
 `,
+					Flags: []cli.Flag{
+						cli.BoolFlag{
+							Name:  "force, f",
+							Usage: "forcefully proceed with replacing peer state with the exported state, without prompting",
```

Comment:
There is a prompt on [line 382](https://github.com/ipfs/ipfs-cluster/pull/553/files#diff-3545a30d0b760577e1788f02eea3d2e0R382). Should that be removed?

Reply:
no, that prompt appears if `-f` flag is false